.lintstagedrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "web/*.php": [
+    "./vendor/bin/php-cs-fixer fix",
+    "./vendor/bin/phpstan analyse --memory-limit=1G"
+  ]
+}
```

Comment:
Since we're calling `ddev exec npx lint-staged` in our `pre-commit` hook, we're already inside the DDEV web container. We have PHP, etc.